app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
```

Comment:
Also better to check that len(parts) is equal to 3
Also check that cmd is "mv" and source path is not the same as destination_path